pages/CheckingTable.js

```diff
@@ -21,8 +17,8 @@ class CheckingPageTable extends AccountTable {
     async changeCheckingTransactionsTablePrevious(){
         await this.checkingTransactionsTablePreviousButton.click()
     }
-    async activateSwitchChecking (){
-        await this.switchChecking.click()
+    async activateSwitchSaving (accountName){
```

Comment:
Why is a method named activateSwitchSaving() this present in the CheckingTable class?